devops/openshift/queue-consumers-deploy.yml

```diff
@@ -0,0 +1,213 @@
+                - name: ZEEBE_AUTHORIZATION_SERVER_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_AUTHORIZATION_SERVER_URL_KEY}
+                      name: ${QUEUE_CONSUMERS_SECRET_NAME}
+                - name: BYPASS_CRA_INCOME_VERIFICATION
+                  value: ${BYPASS_CRA_INCOME_VERIFICATION}
+              image: ${IMAGE_REGISTRY}/${BUILD_NAMESPACE}/${IMAGE_STREAM_TAG}
+              imagePullPolicy: Always
+              name: ${NAME}
+              ports:
+                - containerPort: "${{PORT}}"
+              resources:
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+              readinessProbe:
+                failureThreshold: 10
+                httpGet:
+                  path: /
+                  port: "${{PORT}}"
+                initialDelaySeconds: 40
+                periodSeconds: 30
+                timeoutSeconds: 30
+              livenessProbe:
+                failureThreshold: 10
+                httpGet:
+                  path: /
+                  port: "${{PORT}}"
+                  scheme: HTTP
+                initialDelaySeconds: 10
+                periodSeconds: 30
+                timeoutSeconds: 15
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      name: ${NAME}
+    spec:
+      ports:
+        - name: http
+          port: 80
+          targetPort: ${{PORT}}
+      selector:
+        deploymentconfig: ${NAME}
+      type: ClusterIP
+  - apiVersion: route.openshift.io/v1
+    kind: Route
+    metadata:
+      name: ${NAME}
+    spec:
+      host: ${HOST_NAME}
+      path: ${PATH}
+      tls:
+        insecureEdgeTerminationPolicy: Redirect
+        termination: edge
+      to:
+        kind: Service
+        name: ${NAME}
+parameters:
+  - name: NAME
+    required: true
+  - name: PROJECT
+    value: sims
+  - name: SERVICE_NAME
+    value: queue-consumers
+  - name: CPU_LIMIT
+    value: "1.0"
+  - name: MEMORY_LIMIT
+    value: "2000M"
+  - name: CPU_REQUEST
+    value: "0.5"
+  - name: MEMORY_REQUEST
+    value: "1000M"
+  - name: REPLICAS
+    value: "1"
+  - name: DB_SERVICE
+    value: patroni-master
+  - name: DB_SECRET_NAME
+    value: patroni-creds
+  - name: DB_USERNAME_KEY
+    value: superuser-username
+  - name: DB_PASSWORD_KEY
+    value: superuser-password
+  - name: DATABASE_NAME_KEY
+    required: true
+  - name: IMAGE_STREAM_TAG
+    required: true
+  - name: BUILD_NAMESPACE
+    required: true
+  - name: IMAGE_REGISTRY
+    value: image-registry.openshift-image-registry.svc:5000
+  - name: NODE_ENV
+    value: "production"
+  - name: BUILD_TAG
+    value: "0"
+  - name: QUEUE_CONSUMERS_SECRET_NAME
+    require: true
+  - name: ATBC_LOGIN_ENDPOINT
+    require: true
+  - name: ZEEBE_ADDRESS_KEY
```

Comment:
You can safely remove the required: true property for parameters that have value property, does not need to be specifically mentioned to be required:true.

Also i can see in our workspace we have "require: true" in some of our YAML files, the correct format is "required: true"

We can fix it.

Reply:
Please verify the above comment for the below YAML's too